### PR TITLE
create tab component

### PR DESCRIPTION
**Pull request by MaxenceMottard**

### CODE REVIEW

File: src/components/QTabs/QTabs.vue [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  element.classList.add('q-tab--active');
+  element.style.setProperty('background-color', props.activeBackgroundColor);
+  element.style.setProperty('color', props.activeTextColor);
+
+  emit('selectIndex', tabIndex);
+};
+
+const selectItemFromIndex = (index) => {
+  const children = Array.from(root.value.children);
+  const item = children.find((el) => el.attributes['data-tab-index'].value === index);
```
Comment (MaxenceMottard) [inline src/components/QTabs/QTabs.vue:64]: Can you explain me why do I use `===' ?

Comment (MaxenceMottard) [reply src/components/QTabs/QTabs.vue:64]: So I'll remove it

File: src/main.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { createApp } from 'vue';
 import App from '@/App.vue';
 import components from '@/components';
-import '@/index.css';
+import './index.css';
```
Comment (MrZyr0) [inline src/main.js:4]: 😢

**Review by MrZyr0 — APPROVED** [review]

**Review by ChxGuillaume — CHANGES_REQUESTED** [review]
Make color styling reactive

**Review by ChxGuillaume — APPROVED** [review]

**Pull request merged**